include/krml/internal/target.h

```diff
@@ -151,202 +167,202 @@ inline static int32_t krml_time(void) {
   } while (0)
 
 #if defined(_MSC_VER) && _MSC_VER < 1900
-#  define KRML_HOST_SNPRINTF(buf, sz, fmt, arg) _snprintf_s(buf, sz, _TRUNCATE, fmt, arg)
+#  define KRML_HOST_SNPRINTF(buf, sz, fmt, arg)                                \
+    _snprintf_s(buf, sz, _TRUNCATE, fmt, arg)
 #else
 #  define KRML_HOST_SNPRINTF(buf, sz, fmt, arg) snprintf(buf, sz, fmt, arg)
 #endif
 
 #if defined(__GNUC__) && (__GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ > 4))
 #  define KRML_DEPRECATED(x) __attribute__((deprecated(x)))
-#elif  defined(__GNUC__)
+#elif defined(__GNUC__)
 /* deprecated attribute is not defined in GCC < 4.5. */
 #  define KRML_DEPRECATED(x)
 #elif defined(_MSC_VER)
 #  define KRML_DEPRECATED(x) __declspec(deprecated(x))
 #endif
 
 /* Macros for prettier unrolling of loops */
-#define KRML_LOOP1(i, n, x) { \
-  x \
-  i += n; \
-}
+#define KRML_LOOP1(i, n, x)                                                    \
+  { x i += n; }
 
-#define KRML_LOOP2(i, n, x) \
-  KRML_LOOP1(i, n, x) \
+#define KRML_LOOP2(i, n, x)                                                    \
```

Comment:
What's the reason for these spaces?

Reply:
same, this is clang-format's default settings